lib/github_core.ml

```diff
@@ -330,6 +332,18 @@ module Make(CL : Cohttp_lwt.Client) = struct
       | Error of error
     and 'a t = state -> (state * 'a signal) Lwt.t
 
+    let string_of_message message =
+      message.Github_t.message_message^
```

Comment:
The ugliness of this line just pains me into overload :-) `message...message_message`.  Not fixable for 1.0...